src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -466,7 +466,7 @@
       (fn [{:keys [success? status-code get-parsed-response]}]
         (swap! state assoc :user-email (get-in (get-parsed-response) ["userInfo" "userEmail"]))
         (cond
-          (= status-code 502)
+          (contains #{0 502} status-code)
```

Comment:
sorry, that's what I meant.. I think I'll leave the `contains?` just because it looks a little clearer to me